config/config-server/src/main/resources/upgrades/114.xsl

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<!--
+  ~ Copyright 2018 ThoughtWorks, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
```

Comment:
There should be a test (around `GoConfigMigrationIntegrationTest`) for migration. And also a test that you can now have an environment variable with `${$...}` after the migration. To test the XSD changes.

Reply:
@arvindsv is there any reason to add a migration test? This concept doesn't exist so the migration won't touch the xml except to update the schema version.... or am I missing something?

Reply:
If I wrote it, it would be to make sure that:

1. The migration keeps the content the same.
2. Except for the update of the schema version.

It would be a test for the XSL.